### DISCUSSION TITLE
Make it clear that the Minimum SDK Version is 26 (Oreo) or Android 8.0+

### DISCUSSION THREAD

**Discussion by Unknown78** [Alex313031/Thorium-Android#68] [Ideas]
I get the quick info of it within the details tab of VirusTotal scan result.

For example in the current release:
https://github.com/Alex313031/Thorium-Android/releases/tag/M126.0.6478.246

Apk info of `Thorium_Public_arm32.apk`:
https://www.virustotal.com/gui/file/8c5f7a56f9fb0c65b620b42336616846084f1e8977c3fae7f7428c7389dc4b7c/details

Apk info of `SystemWebView_arm32.apk`:
https://www.virustotal.com/gui/file/e179db51d8e047c98c33f5054b476df14efb107a09c9f5dc8d2484aef35dea3a/details

---

I would suggest to put it in the `readme,md` file that the Minimum SDK Version is 26 (Oreo) or Android 8.0+

I'd really like to see more support on the lower end, but the devs is busy, I just need be more thankful at the moment.

**Comment by gz83** [comment]
Thank you for your suggestion. I believe that, unless otherwise specified, all other unmodified parts are consistent with the upstream code.

In other words, these users can consider checking announcements from Google and viewing the code on Chromium Code Search to find out which operating systems the browser supports.
